caffe2/onnx/backend.cc

```diff
@@ -189,6 +189,17 @@ float OnnxAttributes::get(const std::string& key) const {
   return value;
 }
 
+template <>
+std::string OnnxAttributes::get(const std::string& key) const {
```

Comment:
overloading with return type is dangerous. can we change the existing mechanism and use i (int), f (float), s (string), etc?

Reply:
@houseroad can you elaborate on what changes you want to see here? Overloading with return type is the status quo for` OnnxAttributes::get()`, perhaps refactoring this mechanism should go in a new PR?